dist/conf/logback.xml

```diff
@@ -18,16 +18,14 @@ You may obtain a copy of the License at
 
     <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
         <file>${log.dir:-.}/${log.name:-pravega}.log</file>
-        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+        <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
             <!-- daily rollover. Make sure the path matches the one in the file element or else
              the rollover logs are placed in the working directory. -->
             <fileNamePattern>${log.dir:-.}/%d{YYYY-MM-dd, aux}/${log.name:-pravega}_%d{yyyy-MM-dd_HH}.%i.log.gz</fileNamePattern>
-
-            <timeBasedFileNamingAndTriggeringPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedFNATP">
-                <maxFileSize>10MB</maxFileSize>
-            </timeBasedFileNamingAndTriggeringPolicy>
-            <!-- keep 30 days' worth of history -->
+            <!-- each file should be at most 10MB, keep 30 files worth of history, but at most 10GB -->
```

Comment:
Could you confirm that the folder pattern configured in line 24 works with `SizeAndTimeBasedRollingPolicy` ?

Reply:
`SizeAndTimeBasedRollingPolicy` is a new policy that replaces the deprecated policy `SizeAndTimeBasedFNATP`. I don't see why FileNamePattern should be impacted by the   API change.

Reply:
I've tested this and confirm that new policy `SizeAndTimeBasedRollingPolicy` works with the folder pattern configured.